File: python/1512_number_of_good_pairs.py
class Solution:
    def numIdenticalPairs(self, nums: list[int]) -> int:
        nums_len:int = len(nums)
        pairs:int = 0

        for i in range(0, nums_len):
            for j in range(i + 1, nums_len):
                if nums[i] == nums[j]: pairs += 1
        return pairs

File: javascript/1512_number_of_good_pairs.js
/**
 * @param {number[]} nums
 * @return {number}
 */
function numIdenticalPairs(nums) {
    let numPairs = 0;
    for (let i = 0; i < nums.length; i++) {
        for (let j = i + 1; i < j && j < nums.length; j++) {
            console.log(nums[i], nums[j]); 
            if (nums[i] === nums[j]) {
                numPairs++;
            }
        }
    }   
    return numPairs;
};
